### PR TITLE
WIP: Add documentation to root/socialnet/includes/sn_core_comments.php

### PR DESCRIPTION
**Pull request by senky**
Adding phpDocumentor2 comments before each function, class and method in root/socialnet/includes/sn_core_comments.php

### CODE REVIEW

File: root/socialnet/includes/sn_core_comments.php [hunk 1]
```diff
@@ -1,30 +1,85 @@
...
+ * Core Comments class
+ *
+ * @package phpBB-Social-Network
+ */
 class sn_core_comments
 {
+	/**
+	 * $p_master
+	 *
+	 * @todo add proper description
```
Comment (senky) [inline root/socialnet/includes/sn_core_comments.php]: Please, tell me how to describe `$p_master`.

File: root/socialnet/includes/sn_core_comments.php [hunk 2]
```diff
@@ -1,30 +1,85 @@
...
+	 *
+	 * @var object $p_master
+	 */
 	var $p_master = null;
+
+	/**
+	 * Names of loaded modules
```
Comment (senky) [inline root/socialnet/includes/sn_core_comments.php:39]: I am not sure about this one.

File: root/socialnet/includes/sn_core_comments.php [hunk 3]
```diff
@@ -1,30 +1,85 @@
...
+	 *
+	 * @var array $modulesName
+	 */
 	var $modulesName = array();
+
+	/**
+	 * IDs of loaded modules
```
Comment (senky) [inline root/socialnet/includes/sn_core_comments.php:46]: I am not sure about this one.

File: root/socialnet/includes/sn_core_comments.php [hunk 4]
```diff
@@ -93,6 +154,15 @@ public function add($module, $module_id, $poster, $text)
 
 	/**
 	 * Delete a comment
+	 *
+	 * @access	public
+	 *
+	 * @param 	string|array	$module		module name
+	 * @param 	int		$cmt_id		ID of comment or module
+	 * @param 	bool		$cmt_one	defines, if comment to be deleted will be identified according to comment_id or comment_module_id
```
Comment (senky) [inline root/socialnet/includes/sn_core_comments.php:160]: Please, verify if I am describing it right (I just guess according to code).

File: root/socialnet/includes/sn_core_comments.php [hunk 5]
```diff
@@ -112,7 +182,26 @@ public function del($module, $cmt_id, $cmt_one = true)
...
+	 *
+	 * @access	public
+	 *
+	 * @param 	string|array	$module		module name
+	 * @param 	string		$classPrefix	prefix for CSS class
+	 * @param 	int		$module_id	ID of module
+	 * @param 	int		$cmt_id		ID of comment
+	 * @param 	int|bool	$limit		limit of comments to retrieve
+	 * @param 	bool		$only_comments	defines, if other stuff around comments will be retrieved as well
+	 *
```
Comment (senky) [inline root/socialnet/includes/sn_core_comments.php:193]: Please check description of parametres above.

File: root/socialnet/includes/sn_core_comments.php [hunk 6]
```diff
@@ -241,6 +343,22 @@ public function getField($module, $cmt_id, $field)
 		return $field;
 	}
 
+	/**
+	 * Get posters? See todo!
+	 *
+	 * @todo add all descriptions
```
Comment (senky) [inline root/socialnet/includes/sn_core_comments.php:347]: I really need help here. Please, tell me description of the function, `$with_me` parametre and return array.

File: root/socialnet/includes/sn_core_comments.php [hunk 7]
```diff
@@ -280,6 +407,15 @@ function _moduleName($module)
 		return $moduleName;
 	}
 
+	/**
+	 * Returns module ID according to module name
```
Comment (senky) [inline root/socialnet/includes/sn_core_comments.php:409]: I am not sure about description and return description.

File: root/socialnet/includes/sn_core_comments.php (hunk 1 above)
Comment (Culprit-CZ) [inline root/socialnet/includes/sn_core_comments.php]: `$p_master` is the master object
`$p_master` is the creator this object

**Comment by senky** [comment]
@Culprit-CZ and @Kamahl19, please, check my comments AND whole file for other problems, so that we can finish this file.

**Pull request closed without merging**